docs/current_docs/api/interfaces.mdx

```diff
@@ -4,13 +4,8 @@ slug: /api/interfaces
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
 # Interfaces
 
-:::important
-The information on this page is only applicable to Go and TypeScript SDKs. Interfaces are not currently supported in the Python SDK.
```

Comment:
Since we have SDKs for Java and PHP now, do we need to keep this and reference those SDKs as not implemented?

@eunomie I'm not sure the Java SDK has support for interfaces?

@dragoonis does the PHP SDK have support for interfaces?

Reply:
No, they don't support it. Thanks for the heads up, I can bring it back.

Reply:
Done